src/android_utils.py

```diff
@@ -370,3 +370,43 @@ def listdir(path=None):
 
 def apply_android_workarounds():
     _android11_ext_storage_workarounds()
+
+
+class StartupState:
+    FIRST_TIME = 1
+    USB = 2
+    NETWORK = 3
+    _usb_content_flag = "usb_content_flag"
```

Comment:
Here is a suggestion for the name:
```suggestion
    _usb_content_flag_filename = "usb_content_flag"
```

Reply:
This filename shouldn't be mixed with the states. Please move it to a constant outside.